src/vs/base/browser/ui/scrollbar/scrollableElement.ts

```diff
@@ -373,6 +373,9 @@ export abstract class AbstractScrollableElement extends Widget {
 	}
 
 	private _onMouseWheel(e: StandardWheelEvent): void {
+		if (e.browserEvent?.defaultPrevented) {
+			return;
+		}
```

Comment:
What's this change for? Worried about regressions

Reply:
xterm correctly calls .preventDefault() is a scroll event gets handled by the terminal, but this wasn't respected by the editor. This resulted in scrolling within the terminal when it's shown in the inline peek to scroll the editor when it should instead have just scrolled the terminal.

I'm guessing this is the first time we have something in the editor that manually handles scrolling, so this check was missing.